sdcm/cluster.py

```diff
@@ -5161,7 +5162,8 @@ def start_scylla_monitoring(self, node):
             -s `realpath "{self.monitoring_conf_dir}/scylla_servers.yml"` \
             -n `realpath "{self.monitoring_conf_dir}/node_exporter_servers.yml"` \
             {scylla_manager_servers_arg} \
-            -d `realpath "{self.monitoring_data_dir}"` -l -v master,{self.monitoring_version} -b "-web.enable-admin-api" \
```

Comment:
also what is `--query.max-samples` and how it's related to this commit/PR ?

Reply:
We should set the retention during the setup and not only when restoring because:
1. Sometime we keep monitors for longer periods.
2. Sometimes we/others restore monitors without our code of restoring monitor.

Reply:
@fruch you suggest to set also 100y? i think it is too much